desktop/electron-builder.json

```diff
@@ -41,10 +41,25 @@
 		"artifactName": "wordpress.com-macOS-dmg-${version}-${arch}.${ext}"
 	},
 	"win": {
-		"target": {
-			"target": "nsis",
-			"arch": [ "x64", "ia32" ]
-		}
+		"icon": "./resource/icons/icon.ico",
+		"target": [
+			{
+				"target": "nsis",
+				"arch": [ "x64", "ia32" ]
+			},
+			{
+				"target": "appx",
+				"arch": [ "x64", "ia32" ]
```

Comment:
Do we need both an x64 and ia32 build of the appx, or is a single ia32 installer sufficient? (Will work on both 32-bit and 64-bit systems). Do we submit both for Simplenote?

Reply:
I'm not certain if we need to, we do submit both for Simplenote so I was following that here as well.

Reply:
Makes sense - let's go with whatever we're doing for Simplenote currently.